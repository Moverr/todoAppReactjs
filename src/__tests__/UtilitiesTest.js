import React from 'react';
// import { expect } from 'chai';
import { shallow,render } from 'enzyme';
import renderer from 'react-test-renderer';
import SearchButton from '../Utilities';


const Enzyme = require('enzyme');
const EnzymeAdapter = require('enzyme-adapter-react-16');

// Setup enzyme's react adapter
Enzyme.configure({ adapter: new EnzymeAdapter() });
describe("Main component", () => {

  let headerTitle;
  let wrapper;
  beforeAll(() => {
   
    headerTitle = "TODO APPLICATION";
    wrapper = shallow(<SearchButton name={"testinput"} value={"testvalue"}  callback={testChange}/>);

});

  const testChange = (value)=>{
    return value;
  }

  it('render correctly text component', () => {  
    // const TextInputComponent = renderer.create(<SearchButton />).toJSON();
// let  seearchButtion = renderer.create(<SearchButton/>).toJSON();


    // expect(seearchButtion).toMatchSnapshot();
    expect(wrapper).toMatchSnapshot();
    
});

  /*
it('Header with title', () => {
    expect(wrapper.find('h1').text()).to.equal(headerTitle);
  });

  
it('Header with title', () => {
    expect(wrapper.find('h1').text()).to.not.equal('  WRONG HEADER');
  });
  
*/
});
