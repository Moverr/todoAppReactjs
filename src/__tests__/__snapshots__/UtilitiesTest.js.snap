// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component render correctly text component 1`] = `ShallowWrapper {}`;
